rocketpool/watchtower/submit-rewards-tree-rolling.go

```diff
@@ -391,39 +389,35 @@ func (t *submitRewardsTree_Rolling) getTrueRewardsIntervalSubmissionSlot(targetS
 }
 
 // Checks to see if an existing rewards file is still valid and whether or not it should be regenerated or just resubmitted
-func (t *submitRewardsTree_Rolling) isExistingFileValid(rewardsTreePath string, intervalsPassed uint64, nodeAddress common.Address, isInOdao bool) (rprewards.IRewardsFile, []byte, bool, bool) {
+func (t *submitRewardsTree_Rolling) isExistingFileValid(rewardIndex uint64, intervalsPassed uint64, nodeAddress common.Address, isInOdao bool) (*rprewards.LocalRewardsFile, bool, bool) {
```

Comment:
The new signature simply returns a LocalFile[IRewardsFIle] instead of the byteslice and IRewardsFile, since the new type encapsulates both.

I also replaced rewardsTreePath with rewardIndex to make it a bit more generic.